libs/s25main/ai/aijh/AIPlayerJH.cpp

```diff
@@ -977,13 +977,8 @@ MapPoint AIPlayerJH::FindPositionForBuildingAround(BuildingType type, const MapP
             foundPos = FindBestPosition(around, AIResource::Ironore, BuildingQuality::Mine, searchRadius);
             break;
         case BuildingType::GraniteMine:
-            if(!ggs.isEnabled(
-                 AddonId::INEXHAUSTIBLE_GRANITEMINES)) // inexhaustible granite mines do not require granite
-                foundPos = FindBestPosition(around, AIResource::Granite, BuildingQuality::Mine, searchRadius);
-            else
-                foundPos = SimpleFindPosition(around, BuildingQuality::Mine, searchRadius);
+			foundPos = FindBestPosition(around, AIResource::Granite, BuildingQuality::Mine, searchRadius);
```

Comment:
Only the indentation is off:
```suggestion
          foundPos = FindBestPosition(around, AIResource::Granite, BuildingQuality::Mine, searchRadius);
```